common/src/main/java/earth/terrarium/adastra/api/events/AdAstraEvents.java

```diff
@@ -43,6 +47,26 @@ static boolean post(ServerLevel level, LivingEntity entity) {
         }
     }
 
+    @FunctionalInterface
+    public interface EntityHasOxygenEvent {
+        Optional<Boolean> tick(Entity entity, boolean hasOxygen);
```

Comment:
Something like `hasOxygen` would make more sense than `tick`. This event is only ever posted when checking oxygen, not based on any tick.

Reply:
oops, I forgot to change it.